lib/containers/runtime.pm

```diff
@@ -0,0 +1,142 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Runtime interface for container management
+# Maintainer: qac team <qa-c@suse.de>
+
+package containers::runtime;
+use Mojo::Base -base;
+use testapi;
+use Test::Assert 'assert_equals';
+
+has runtime => undef;
+
+sub _rt_assert_script {
```

Comment:
```suggestion
sub _assert_script_run {
```

Reply:
i had cut the _run to make it a bit shorter. i changed it to `_rt_assert_script_run` to keep consistency with the rest and indicate that it is very specific to runtime.

Reply:
to keep consistency I would go to do same renaming for other methods too  `_script_output`  etc. 
`$self->_asssert_script_run` is clearly show you that you dealing not with regular `assert_script_run` . Cutting `_run` not very good idea when you creating wrapper method because it creates confusion if it is actual wrapper or something custom

Reply:
i hope it is fine as it is now.

Reply:
No. It still does have the `rt_` prefix.

Reply:
i added `_run`. i think then @asmorodskyi proposed to get rid of `_rt` to keep the consistency with his initial request. i want to keep `_rt` which is short for _runtime to indicate that this is owned by runtime specifically. The request for this wrappers came from another request. As those changes took place and they approved by the initial reviewer i dont think i should have move on with any change on the naming anyway. i am gonna leave it as it is.